ecosystem/sep-0024.md

```diff
@@ -869,6 +870,7 @@ Name | Type | Description
 `id` | string | (optional) The id of the transaction.
 `stellar_transaction_id` | (optional) string | The stellar transaction id of the transaction.
 `external_transaction_id` | (optional) string | The external transaction id of the transaction.
+`lang` | string | (optional) Defaults to `en`. Language code specified using [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1).
```

Comment:
```suggestion
`lang` | string | (optional) Defaults to `en`. Language code specified using [RFC 4646](https://datatracker.ietf.org/doc/html/rfc4646) which means it can also accept locale in the format `en-US`.
```

Reply:
modified.